cli/pcluster/config/validators.py

```diff
@@ -237,6 +247,13 @@ def dcv_enabled_validator(param_key, param_value, pcluster_config):
         if get_partition() not in get_supported_dcv_partition():
             errors.append("NICE DCV is not supported in the selected region '{0}'".format(get_region()))
 
+        if pcluster_config.get_section("dcv").get_param_value("access_from") == "0.0.0.0/0":
```

Comment:
Minor, could we use a meaningful utils constant name like CIDR_ALL_IPS?

Reply:
done